docs/dev-guides/9-xr-details-api.md

```diff
@@ -0,0 +1,142 @@
+---
+id: xr-details-api
+title: xr_details API
+---
+
+An overview of XRPackages and the `manifest.json` file is provided in [the XRPackage Overview](./1-xrpackage-overview.md).
+
+This page details the API of the `xr_details` field, which allows you to provide further details about how a package should be treated by a compatible runtime.
+
+Each heading below corresponds to the _optional_ key in the `xr_details` object of your XRPackage `manifest.json`.
+
+## `aabb`
+
+_Defines the <a href="https://developer.mozilla.org/en-US/docs/Games/Techniques/3D_collision_detection" target="_blank" rel="noopener noreferrer">axis-aligned bounding box</a> for this package._
+
+**Format**: a 2D array with the first inner array defining the minimum point, and second inner array defining the maximum point.
+
+**Example**:
+
+```json
+{
+  "xr_type": "webxr-site@0.0.1",
+  "start_url": "cube.html",
+  "xr_details": {
+    "aabb": [
+      [-1, -1, -1], // min point
+      [1, 1, 1] // max point
+    ]
+  }
+}
+```
+
+## `events`
+
+_Describes events the package can receive from other packages._
+
+**Format**: an Object with keys corresponding to the event names to be received, and values being an object that is `{"type": "string"}`. Currently, the `type` _must_ be `string`.
+
+**Example**: in this example, we state that this package can receive the `reloof` event from other packages, with the payload type being a `string`.
+
+```json
+{
+  "xr_type": "webxr-site@0.0.1",
+  "start_url": "cube.html",
+  "xr_details": {
+    "events": {
+      "reloof": { "type": "string" }
+    }
+  }
+}
+```
+
+## `physics`
+
+_Defines the physics mode for the package._
+
+**Format**: a string (`static`, or `default`), or `null` (no physics &mdash; the default)
+
+**Example**:
+
+```json
+{
+  "xr_type": "webxr-site@0.0.1",
+  "start_url": "cube.html",
+  "xr_details": { "physics": "static" }
+}
+```
+
+## `schema`
+
+_Describes the configurable properties of the package._
+
+**Format**: a `floofiness` Object with keys `type` and `default`. Currently, the `type` _must_ be `string`.
+
+**Example**:
+
+```json
+{
+  "xr_type": "webxr-site@0.0.1",
+  "start_url": "cube.html",
+  "xr_details": {
+    "schema": {
+      "floofiness": {
+        "type": "string",
+        "default": "very"
+      }
+    }
+  }
+}
+```
+
+## `script`
+
+_Defines a worker JavaScript file for GLTF packages._
+
+**Format**: a string that is the filename of the JavaScript script to run.
```

Comment:
Should at least have a placeholder describing how that is loaded (in a Worker), exposing a THREE.js Object bound to the GLTF package.